src/App.js

```diff
@@ -1,11 +1,51 @@
-import React from 'react';
+import React, {useEffect, useState} from "react";
+import axios from "axios";
 import './App.css';
+import './components/PokemonCard';
+import PokemonCard from "./components/PokemonCard";
 
 function App() {
+    const [endpoint, setEndpoint] = useState(['https://pokeapi.co/api/v2/pokemon'])
+    const [pokemonData, setpokemonData] = useState([]);
+
+    useEffect(() => {
+        async function fetchPokemonData()
+        {
+            try
+            {
+                const pokemonData = await axios.get(`${endpoint}`);
+                console.log(pokemonData.data)
+                setpokemonData(pokemonData.data)
+            }
+            catch(e)
+            {
+                console.log(e)
+            }
+        }
+        fetchPokemonData()
+
+    }, [endpoint])
+
   return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
+    <>
+        <nav>
+            <button
+                type='button'
+                disabled={pokemonData.previous === null}
+                onClick={() => setEndpoint(pokemonData.previous)}>
+                Previous
+            </button>
+            <button
+                type='button'
+                disabled={pokemonData.next === null}
+                onClick={() => setEndpoint(pokemonData.next)}>
+                Next
+            </button>
+        </nav>
+        <section>
+            { pokemonData.results && pokemonData.results.map((pokemon) => <PokemonCard url={pokemon.url} key={pokemon.name} /> )}
```

Comment:
Wanneer je de default waarde `undefined` hebt gemaakt, dan kun je het hier wat versimpelen
```suggestion
            { pokemonData && pokemonData.results.map((pokemon) => <PokemonCard url={pokemon.url} key={pokemon.name} /> )}
```